src/components/projects/lean-workflow/ManuscriptPageContainerLW.tsx

```diff
@@ -163,9 +163,18 @@ const ManuscriptPageView: React.FC = () => {
 
   const { setUsers } = useCommentStore()
   const { updateDocument } = useDocStore()
-  const { init: initEditor, setEditorState } = useEditorStore()
+  const { init: initEditor, setEditorState, trackState } = useEditorStore()
   useMemo(() => setUsers(collaboratorsById), [collaboratorsById, setUsers])
   useMemo(() => view && initEditor(view), [view, initEditor])
+
+  const hasPendingSuggestions = useMemo(() => {
+    const { changeSet } = trackState || {}
+    if (changeSet && changeSet.pending.length > 0) {
+      return true
+    }
+    return false
```

Comment:
you could simplify this to `return changeSet && changeSet.pending.length > 0`

Reply:
Done